es/introduction.md

```diff
@@ -121,35 +121,35 @@ Si el proceso de auditoría se añade a una codebase existente, las nuevas solic
 Este Estándar ayuda a profesionales del desarrollo, diseño, de la gestión de equipos empresariales y responsables de políticas a:
 
 * Desarrollar software y políticas de alta calidad para mejorar la prestación de servicios públicos.
-* desarrollar codebases que puedan reutilizarse en distintos contextos y mantenerse en colaboración
-* reducir la deuda técnica y la tasa de fracaso de los proyectos
-* tener un mayor control y capacidad de decisión sobre sus sistemas de IT
-* mejorar las relaciones con los proveedores con un mejor modelo económico
+* Desarrollar codebases que puedan reutilizarse en distintos contextos y mantenerse en colaboración.
+* Reducir la deuda técnica y la tasa de fracaso de los proyectos.
+* Tener un mayor control y capacidad de decisión sobre sus sistemas de IT.
+* Mejorar las relaciones con los proveedores con un mejor modelo económico.
 
-[La Foundation for Public Code](https://publiccode.net/) ayuda a las organizaciones públicas a compartir y adoptar software de código abierto, construir comunidades de desarrolladores sostenibles y crear un ecosistema próspero para el código público. Lo hace a través de la tutela o stewardship de codebases. Para este proceso, los administradores de codebases utilizan el Estándar para el Código Público para asegurarse de que el código que supervisan es de alta calidad y se puede mantener en colaboración.
+[La Foundation for Public Code](https://publiccode.net/) ayuda a las organizaciones públicas a compartir y adoptar software de código abierto, construir comunidades de desarrolladores sostenibles y crear un ecosistema próspero para el código público. Lo hace a través de la tutela o dirección de codebases. Para este proceso, los administradores de codebases utilizan el Estándar para el Código Público para asegurarse de que el código que supervisan es de alta calidad y puede ser mantenido de manera colaborativa.
```

Comment:
Change for: "Lo hace a través de la tutela o stewardship de codebases. Para este proceso, las personas que administran las codebases utilizan el Estándar para el Código Público para asegurarse de que el código que supervisan es de alta calidad y puede ser mantenido de manera colaborativa." How about this one?